### DISCUSSION TITLE
关于夜间模式图文专栏字体颜色的问题

### DISCUSSION THREAD

**Discussion by SteveXu9102** [the1812/Bilibili-Evolved#4918] [综合讨论 / General]
我开夜间模式看到的专栏长这样：

![屏幕截图 2024-09-18 011025](https://github.com/user-attachments/assets/babdddeb-de74-4173-8303-bd21e7adbef6)
https://www.bilibili.com/read/cv38851384/?jump_opus=1

这字体颜色属实让人眼瞎。
但是issues里搜专栏又没注意到相同问题，所以我先来问问看是不是普遍情况。
求大佬解答。

**Comment by Geeyun-JY3** [comment]
夜间模式适配跟不上是常态。
